### PR TITLE
feat: Npm dependencies

### PR DESCRIPTION
**Pull request by Pratik33**
feat: React, React-dom and parcel.js

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,11 +1,15 @@
 {
```
Comment (ruchamahabal) [inline .vscode/settings.json:1]: Remove this file from your PR.

**Review by ruchamahabal — COMMENTED** [review]
Can you also push the create-react-app part on your repo?

**Comment by rishav-sah** [comment]
Your parcel build will be failed because the path is not provided there, in CSS for colors use HEX, and for fonts better use hosted fonts like Google Font API.

**Comment by Pratik33** [comment]
> Your parcel build will be failed because the path is not provided there, in CSS for colors use HEX, and for fonts better use hosted fonts like Google Font API.

Thanks for your suggestion.
I have added the build path.

**Pull request merged**